### PR TITLE
 Fixing checkstyle issues and upgrading job chart

### PR DESCRIPTION
**Pull request by adusumillipraveen**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RPE-1094

### Change description ###

 Fixing checkstyle issues and upgrading job chart

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -7,6 +7,7 @@ def product = "data-extractor"
 def component = "job"
 
 withPipeline(type, product, component) {
+  env.TESTCONTAINERS_RYUK_DISABLED = 'true'
```
Comment (timja) [inline Jenkinsfile_CNP]: what's this do?

Comment (timja) [reply Jenkinsfile_CNP]: > ryuk image helps to remove and cleanup containers.

apparently, what's wrong with it?

File: src/integrationTest/java/uk/gov/hmcts/reform/dataextractor/DbTest.java [hunk 2]
```diff
@@ -50,9 +50,15 @@
                             .build(),
                     insertInto("child")
                             .columns("id", "name", "data", "created_date", "parent_id")
-                            .values(1, "A1", "{\"a1Data\": {\"name\": \"A1\", \"subject\": {\"name\": {\"lastName\": \"A1S\", \"firstName\": \"Harry\"}, \"address\": \"a1Address\" }}}::json", "2019-04-12 23:45:46", 1)
-                            .values(2, "A2", "{\"a2Data\": {\"name\": \"A2\", \"subject\": {\"name\": {\"lastName\": \"A2S\", \"firstName\": \"Harry\"}, \"address\": \"a2Address\" }}}::json", "2019-04-12 23:45:47",1)
-                            .values(3, "B1", "{\"b1Data\": {\"name\": \"B2\", \"subject\": {\"name\": {\"lastName\": \"B1S\", \"firstName\": \"Harry\"}, \"address\": \"b1Address\" }}}::json", "2019-04-12 23:45:46", 2)
+                            .values(1, "A1", "{\"a1Data\": {\"name\": \"A1\", \"subject\": "
```
Comment (timja) [inline src/integrationTest/java/uk/gov/hmcts/reform/dataextractor/DbTest.java]: might make sense to extract to a file? this isn't very readable or maintainable with all the escaping

File: Jenkinsfile_CNP (hunk 1 above)
Comment (adusumillipraveen) [reply Jenkinsfile_CNP]: We have an issue while using test containers on Jenkins, we need to authorise quay repository for ryuk. We were just following some recommendations on issue to test if it resolves. https://github.com/testcontainers/testcontainers-java/issues/700 , apparently it doesn't as we need to authorise basic postgres one as well.

Comment (timja) [reply Jenkinsfile_CNP]: weird as lots of apps here use testcontainers on jenkins

Comment (timja) [reply Jenkinsfile_CNP]: I think this repo does some manual init of test containers in a rule:
https://github.com/hmcts/cmc-claim-store

may be worth having a look, it specifies which image to use I believe

File: src/integrationTest/java/uk/gov/hmcts/reform/dataextractor/DbTest.java [hunk 3]
```diff
@@ -50,9 +54,15 @@
...
+                                    + "{\"name\": {\"lastName\": \"A1S\", \"firstName\": \"Harry\"}, "
```
Comment (timja) [inline src/integrationTest/java/uk/gov/hmcts/reform/dataextractor/DbTest.java]: did you not want to extract this too?

**Pull request merged**